fiesta/apps/events/views/event.py

```
@@ -0,0 +1,324 @@
+    
+    def dispatch(self, request, *args, **kwargs):
+        self.event = get_object_or_404(Event, pk=self.kwargs.get("pk"))
+        return super().dispatch(request, *args, **kwargs)
+    
+
+    def post(self, request, *args, **kwargs):
+        validator = ParticipantValidator()
+        try:
+            price = validator.register_for_event(self.request.user, self.event)
+            Participant.objects.create(
+            created=datetime.datetime.now(),
```

Ruff

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)